unit-tests/log/test-all.py

```diff
@@ -0,0 +1,25 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+from rspy import log, test
+import pyrealsense2 as rs
+import common
+
+
+#############################################################################################
+#
+test.start( f"log all" )
+
+try:
```

Comment:
Why do we need `try`?
LibCI catches an exception as a failure, right? no need for 
```
try:
...
except:
    test.unexpected_exception()
```
on every test

Reply:
If something throws outside a `try` then the whole file crashes and you get a stack trace.
What we're saying here is that we do not expect any of the statements to throw and that, if they do, we'll catch them gracefully and continue on with the other tests.

Reply:
That means we should do it for each test.

Normally I tell others to drop it, it it throws libci will catch, otherwise we will have the same template for each test we write..
What do you think?

Reply:
Like I said: LibCI will "catch" it by crashing the test. You won't even get a count of the number of failed asserts.
If that's acceptable, so be it.
Personally, I think it's clearer to say: "I'm not expecting any of this to throw" as a conscious, verbose, thing.